src/mSupplyMobileApp.js

```diff
@@ -151,6 +152,7 @@ class MSupplyMobileAppContainer extends React.Component {
     } else {
       this.postSyncProcessor.processRecordQueue();
     }
+    dispatch(syncCompleteTransaction());
```

Comment:
This naming doesn't really make sense. Want a way to signal 'after post processing sync', but don't want to pass dispatch to post sync processor